airflow/providers/databricks/operators/databricks.py

```diff
@@ -333,6 +341,8 @@ def __init__(
             self.json['spark_submit_task'] = spark_submit_task
         if pipeline_task is not None:
             self.json['pipeline_task'] = pipeline_task
+        if dbt_task is not None:
+            self.json['dbt_task'] = dbt_task
```

Comment:
If the git source is required, I would move this code  after this line https://github.com/apache/airflow/pull/25623/files#diff-30731826b25d422cb6069898701e34f8ef23b2a32b7782444793a14bc52b0cb1L352 and throw exception if it isn’t set - this will allow to catch misconfigurations early…